python3/ltk/actions/action.py

```diff
@@ -200,6 +222,79 @@ def update_config_file(self, option, value, conf_parser, config_file_name, log_i
             print(e.errno)
             print(e)
 
+    def metadata_wizard(self, fields, set_defaults=False):
```

Comment:
You're a metadata wizard! This code looks great!!

Reply:
Heh, thanks